src/common/math/math-algebra3d.c

```diff
@@ -0,0 +1,88 @@
+#include "common/math/math-algebra3d.h"
+#include "common/math/math.h"
+#include <stdbool.h>
+#include <stdint.h>
+#include <math.h>
+
+/**
+ * Special operations in R3, as they are used often
+ * Cross product only exists in R3 
+ * Rotation matrix transpose is inverse rotation
+ */
+// vector * scalar | vector scaling
+vector3d_t math_vector_scale(float scalar, const vector3d_t *vector)
```

Comment:
It's very obvious within the source file, but when the functions are actually being used its not immediately clear they are 3d vector operations. Would prefer `math_vector3d_scale` or similar

Reply:
This is fine if we are only operating on 3d vectors in the entire code but I assume we aren't (e.g. estimator state vector)